virt/virtual_machines/advanced_vm_management/virt-configuring-mediated-devices.adoc

```diff
@@ -0,0 +1,48 @@
+:_content-type: ASSEMBLY
+[id="virt-configuring-mediated-devices"]
+= Configuring mediated devices
+include::modules/virt-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: virt-configuring-mediated-devices
+
+toc::[]
+
+{VirtProductName} automatically creates mediated devices, such as virtual GPUs (vGPUs), if you provide a list of devices in the `HyperConverged` custom resource (CR).
+
+ifdef::openshift-enterprise[]
+:FeatureName: Automatic configuration of mediated devices
+include::snippets/technology-preview.adoc[]
+endif::[]
+
+[id="prerequisites_virt-configuring-mediated-devices"]
+== Prerequisites
+
+* You installed any required drivers on the nodes where you want to create mediated devices.
+** If you use NVIDIA cards, you link:https://access.redhat.com/solutions/6738411[installed the NVIDIA GRID driver].
+
+* Your GPU device supports GPU passthrough.
```

Comment:
AFAIK, This line referring to "GPU passthrough" is not relevant here, while "Configuring mediated devices". We need to drop this. @vladikr  wdyt ?